README.md

```diff
@@ -50,6 +50,8 @@ looking. You've came to the right place.
     this repository to show your love.
 
 ### Note
+- If within your gitlab CI Code, you have a pause within the script code, this notiction yml code will still run, to fix this, we need to add allow_failure: false, this will mean the job will not run unless told to(see screenshot)
```

Comment:
```suggestion
- If you pause the GitLab CI build, `after_success` still gets triggered resulting in this webhook notification being fired. To work around this, we need to add `allow_failure: false` to the config (see screenshot).
```
Rephrased the sentence.

Reply:
> (see screenshot)

Did you forget to add the screenshot?

Reply:
Screenshot not needed, as it will be different per CI setup.

Reply:
Then why is `(see screenshot)` added?